blog/models.py

```diff
@@ -1,8 +1,11 @@
+"""Description for models for interfacing the DB."""
 from django.db import models
 from django.utils import timezone
 
 
 class Post(models.Model):
+    """Definition of the model."""
```

Comment:
I see that from the line above as it inherits model. Don't refer to implementation details, unless they really matter + don't repeat smth obvious one can notice w/o a docstring.
Better tell me that it's a blog post structure declaration.